server/src/routes.ts

```diff
@@ -8,36 +8,44 @@ type temp = {
   name: string;
 };
 
+//Check's the server is working fine or not
+
 function routes(app: Express) {
   app.get("/healthcheck", (req: Request, res: Response) => res.sendStatus(200));
 
+
```

Comment:
remove this extra line

Reply:
ye buddha mere bicchem me bahut bolta hai